sidhariyal/constellation_names.eng.fab

```diff
@@ -0,0 +1,88 @@
+Aql	"Aquila"	_("கருடன்")

```

Comment:
If this shall be a "traditional" SC, it likely has no modern constellations like Scutum or the others for which you don't provide (wrongly) a Tamil name. Again, the "name" in `_("name")` must contain English transcription and translation if different from the common names. Else Tamil names are provide by translating the common constellation names in Transifex.

Reply:
Understood. The team's primary objective was to make Stellarium accessible to Tamil-speaking Farmers and Stargazers who may not be proficient in English. During our exploration of available options, I observed that simply altering the _("name") parameter addressed our goal effectively. Leveraging the Western Sky Culture as a foundation due to its significant parallels, we proceeded to modify the _("name") to its Tamil equivalent.  To avoid Transifex, can I follow something like Aql	"கருடன்" _("Karutan")?

Reply:
To translate names from the default culture, you use Transifex. This is by far the simplest way to present an almost-identical-to-western/modern SC for your people. This is the usual case for a 21st century living tradition (usually among amateur astronomers) that incorporates the scientifically canonized 88 constellations. All "modern-"`sky cultures use these IAU constellations (regions). 

If having an independent "traditional" SC is important, where maybe 80% of figures is identical, it still must be started as translatable skyculture. No translator on Transifex could translate "கருடன்" into any useful string for English, German, Russian, French, Italian, Spanish etc. without knowing Tamil. (Same goes with other non-European scripts. Sorry, we are usually illiterate here!) If you provide the format `Aql "கருடன்" _("Karutan")`, all Tamils can set the SC display to "original" and see their text in their script. Non-Tamils interested in the SC can use "translated" and see "Karutan" which at least helps in pronunciation. Now, on Transifex, again no translator would know what Karutan means unless he knows Tamil. Understood as proper name, it likely gets copied into all the other languages as-is, maybe further transliterated like "kaрутaн" (?) in Russian. If it is supposed to just mean "Eagle", this information is lost in the process. Therefore, the best format currently would be `Aql "கருடன்" _("Karutan (Eagle)")` [if Karutan means eagle, which I don't know.] 

If you want to present a contemporary SC used by today's amateurs in your region, you would usually create a "modern-Tamil" SC. This would use the 88 IAU constellations and borders (so that you could discuss discoveries with the international community), but may show slightly different lines or artwork. (Maybe slight animal changes, like Buffalo/Taurus.)  Regional specialties, where older traditions are incompatible with Ptolemy's constellations,  could then be also added in the asterisms.  However, when the Tamil SC has not been derived only in the 19th/20th century from European sources, but has an independent much older tradition with links to early oriental astronomy (Ptolemy, Babylon, ...), then all the modern constellations which are not traditional Tamil constellations, like Scutum, Sextans, Pictor, Fornax etc.,  should be removed. The same goes with all star names. The SC files should only include names actually used in that culture, with additional accurate reference links to source books, or provide in the description some information if and why information/data from other cultures is presented. In any case, a good description text is mandatory to be of interest to our users.

Reply:
Noted with thanks, I'll incorporate it.